### DISCUSSION TITLE
success with Orange Pi Zero 2

### DISCUSSION THREAD

**Discussion by zgap118** [mcguirepr89/BirdNET-Pi#444] [Show and tell]
Just a note to say to I've had success with installing BirdNet-Pi on a OrangePi Zero2.

http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html

They're a lot easier to get hold at the moment. USB mic at the moment, but hope to get bluebooth mic working eventually.

The RPi install notes pretty much work identically for the OPi using their Debian images. From my very cursory notes, I think there were just a couple of missing packages. sqlite and python3-venv

👍 

Tom

**Comment by yjeanrenaud** [comment]
digging this out, but would you happen to have some more details? 
I tried with Orangepizero2_3.0.6_debian_bullseye_server_linux5.16.17  nd the installer script gave a success message, but nothing seems to work.

**Comment by lemmin4kainen** [comment]
Hi. @zgap118 please share your experience on how you did it.

**Comment by yjeanrenaud** [comment]
I managed to get it working. will sum it up soon

**Comment by BAM-Team** [comment]
I used Image
Orangepizero2w_1.0.0_debian_bullseye_server_linux6.1.31
This worked for me. I had to repeat the repo Installation 3 times due to possibly network problems(?) . 
But in the end everything is fine now. It performs way better than the RasPi02w :-)
Will provide a better Handout and first test results soon, hopefully...
Kind  regards Carsten

btw.: HW is Orange Pi Zero 2 W 4 GB

**Reply by lemmin4kainen** [reply]
> I used Image Orangepizero2w_1.0.0_debian_bullseye_server_linux6.1.31 This worked for me. I had to repeat the repo Installation 3 times due to possibly network problems(?) . But in the end everything is fine now. It performs way better than the RasPi02w :-) Will provide a better Handout and first test results soon, hopefully... Kind regards Carsten
> 
> btw.: HW is Orange Pi Zero 2 W 4 GB

Hello, can you share the installation instructions?

**Comment by BAM-Team** [comment]
Yes...Quick and dirty german Version is attached 
[Birdnet OrangePI02w Installation.pdf](https://github.com/user-attachments/files/18162893/Birdnet.OrangePI02w.Installation.pdf)
here - I will provide an english version asap. 

Kind regards Carsten

**Reply by BAM-Team** [reply]
btw.: I´m using Boya BY-LM40 microphone. It has an excellent sensitivity & build in Soundcard w. direct USB connector

**Reply by BAM-Team** [reply]
Here is an english version. Quick and dirty, too :-)
[Github Birdnet OrangePI02w Installation EN.pdf](https://github.com/user-attachments/files/18168847/Github.Birdnet.OrangePI02w.Installation.EN.pdf)

I am not a linux pro - additions and improvements are welcome!
